main/midnight/src/tme/scenarios/ddr/ddr_character.cpp

```diff
@@ -884,7 +885,7 @@ namespace tme {
             riders.Total(total);
         
         stronghold_total -= diff ;
-        stronghold->Total(MAX(stronghold_total,0));
```

Comment:
I thought it would be ok to use std::max like other parts of code instead of MAX macro that was not defined. Same for MIN later down the code.